Add Swift sample and remove objc sample

x-0o0 opened the pull request:
- Added Swift sample
- Added FAQ bot features
- Added Sendbird UIKit dependency
- Modified README

Commit:
Add Swift sample and remove objc sample

mininny commented on QuickStart/AppDelegate.swift:
```diff
@@ -0,0 +1,47 @@
+        
+//        let center = UNUserNotificationCenter.current()
+//
+//        application.registerForRemoteNotifications()
```
Please remove the app id. 
And I think we need to uncomment lines 24~26 and enable remote notifications

x-0o0 replied:
done

mininny commented on QuickStart/SettingsViewController.swift:
```diff
@@ -0,0 +1,59 @@
+    
+    @IBAction func showOpenSourceLicense(_ sender: Any) {
+        
+    }
```
I don't think these will have any implementation; can we remove them here and in the storyboard?

x-0o0 replied:
done

mininny commented on QuickStart/LoginViewController.swift:
```diff
@@ -0,0 +1,48 @@
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
```
I don't think we need these :)

x-0o0 replied:
done!

mininny commented on QuickStart/InboxViewController.swift:
```diff
@@ -0,0 +1,221 @@
+        }
+    }
+    
+    func handleEmptyResultView(shouldShow: Bool, error: Error?) {
```
I think this should run on main queue. Can you add `SBDSKMain. setCompletionHandlerDelegateQueue` in the AppDelegate?

x-0o0 replied:
Already added!

mininny commented on QuickStart/InboxViewController.UITableViewDelegate.swift:
```diff
@@ -0,0 +1,38 @@
+    
+    func tableView(_ tableView: UITableView, estimatedHeightForHeaderInSection section: Int) -> CGFloat {
+        return tickets.count == 0 && hasMoreTickets == false ? view.bounds.height : 0
+    }
```
We don't need this anymore because we don't have header in the tableview.

x-0o0 replied:
done

mininny commented on QuickStart/InboxTicket/InboxTicketTableViewCell.swift:
```diff
@@ -0,0 +1,76 @@
+import UIKit
+import SendBirdDesk
+
+class InboxTicketTableViewCell: UITableViewCell {
+    static let identifier: String = "InboxTicketTableViewCell"
```
Can't we just use `String(describing: Self.self)`

x-0o0 replied:
done

mininny commented on QuickStart/FAQBot/FAQMessageView.swift:
```diff
@@ -0,0 +1,184 @@
+        } else {
+            imageView.isHidden = true
+        }
+        
+        
```
Can we clean up these double newlines? I see many of them in the code

x-0o0 replied:
done

mininny commented on QuickStart/FAQBot/FAQBotMessageCell.swift:
```diff
@@ -0,0 +1,66 @@
+        faqData.faqResults.forEach { result in
+            let faqMessageView = FAQMessageView(result: result)
+            faqMessageView.frame = CGRect(x: 0, y: 0, width: 244, height: 82)
+            faqMessageView.delegate = self
+            faqStackView.addArrangedSubview(faqMessageView)
+        }
+        faqStackView.updateConstraints()
+    }
+}
+
+extension FAQBotMessageCell: FAQMessageViewDelegate {
+    func didSelect(_ question: String) {
+        guard let faqFileID = faqData?.faqFileId else { return }
+        delegate?.didSelectQuestion(faqFileID, question: question)
+    }
```
Could we have just ChatViewController implement this FAQMessagViewDelegate, and only FAQMessageView calls this delegate method? It feels unnecessary to have FAQBotMessageCell implement this delegate and relay it to ChatViewController again as a delegate.

x-0o0 replied:
The, FAQMessageView must have FAQ data which has all information of FAQ in the message such as a list of FAQ results and FAQ file ID. However, IMO, FAQMessageView is a view for FAQ result only, so it just needs a FAQ result which is included in FAQ data, not all information of FAQ.

mininny commented on QuickStart/ChatViewController.swift:
```diff
@@ -0,0 +1,68 @@
+        super.viewDidLoad()
+        
+        // Register XIB: SystemBotMessageCell
+        let systemBotNib = UINib(nibName: "SystemBotMessageCell", bundle: nil)
```
Use class identifier here as well?

x-0o0 replied:
done

mininny reviewed (changes requested):
Looks great overall! Just some comments. 
Welcome to yet another swift project 🥳

tezpark reviewed (commented):
* If iOS 13 exception handling is not done, build error occurs in `UIScene` related codes
  * need to add`@available(iOS 13.0, *)` annotation
* empty space between `messageInputView` and `messageCell`
* Crash when attaching a file
* It would be better to remove the edit button from the ChannelSettings. (I think it would be better to remove the channel settings button.)
* After the Leave channel action, the channel does not disappear from the list. (Crash when clicking the channel)

x-0o0 commented:
> * If iOS 13 exception handling is not done, build error occurs in `UIScene` related codes
>   
>   * need to add`@available(iOS 13.0, *)` annotation
> * empty space between `messageInputView` and `messageCell`
> * Crash when attaching a file
> * It would be better to remove the edit button from the ChannelSettings. (I think it would be better to remove the channel settings button.)
> * After the Leave channel action, the channel does not disappear from the list. (Crash when clicking the channel)

- Removed Scene delegate because it's not needed.
- Removed channel settings button from channel header
- Fixed issue on empty space between messages and textfield
- the inboxVC doesn't receive event of channel list. I'll add when the all platforms' design renewal

mininny commented on QuickStart/AppDelegate.swift:
```diff
@@ -0,0 +1,27 @@
+    
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // Intialize Chat and Desk SDK
+        SBDMain.initWithApplicationId("YOUR_APP_ID")
```
This will run without error. How about `<#YOUR_APP_ID#>` instead of actual string?

x-0o0 replied:
done!

Commit:
Modified app ID regarding code
